src/components/Result.tsx

```diff
@@ -53,22 +54,44 @@ function generateCreatorElements(item: CMMStudy) {
   return creators;
 }
 
+// Randomize array using Durstenfeld shuffle algorithm
+function shuffleArray<T>(array: Array<T>) {
+  if (array) {
+    for (let i = array.length - 1; i > 0; i--) {
+      const j = Math.floor(Math.random() * (i + 1));
+      [array[i], array[j]] = [array[j], array[i]];
+    }
+  }
+  return array;
+}
+
```

Comment:
This should probably go in a utilities module

Reply:
Thanks. I think that's a good idea so I moved it.